### PR TITLE
[WIP] feat: add versioning to Specs

### PR DESCRIPTION
**Pull request by cstrnt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
There is no way to version Specs

**What is the new behavior (if this is a feature change)?**
Specs can now be versioned and share code more easily

**Additional info:**

### CODE REVIEW

File: schemas/fig.d.ts [hunk 1]
```diff
@@ -198,6 +198,9 @@ declare namespace Fig {
     parserDirectives?: {
       flagsArePosixNoncompliant?: boolean;
     };
+
+    getVersionCommand?: string;
```
Comment (sullivan-sean) [inline schemas/fig.d.ts:202]: Is there a reason this is on `Subcommand` as opposed to `Spec`?

**Review by sullivan-sean — COMMENTED** [review]
Looks great 🙌  Only change that might be nice is an update to the `README.md` explaining a bit of how versioning works for future contributors

**Comment by Nutlope** [comment]
Hey @cstrnt, heads up that @QuiiBz and I merged a ton of specs in the past 24h so re-pulling from master would be great before merging. Also give me a heads up when this is merged so I can update the docs accordingly. Great work man!

**Review by fedeci — COMMENTED** [review]
Why not using workspaces (yarn) and make each spec a package that can be downloaded separately from npm. 
Benefits:
- Modularity
- A user could just install what needs

Not strictly related to this PR, but may be good to join the two things.

**Pull request closed without merging**